src/plays/age-calculator-by-react/Calculator.jsx

```diff
@@ -0,0 +1,19 @@
+import Header from './Header';
+import Input from './Input';
```

Comment:
Consider using more descriptive names for the imported components. For example, instead of `Header, Input, and Output,` you could use `CalculatorHeader, AgeInput, and AgeOutput` respectively. 

This improves clarity and understanding of the component's purpose.